core/performance_test/peakflops/common.h

```diff
@@ -3,6 +3,7 @@
 
 #include <gtest/gtest.h>
 
+// NOTE: The user may have to override these for a given platform
```

Comment:
Should we add a configure option for this?

Reply:
Right now that's why they're wrapped in `#ifndef` so the user can specify `-DEXPECTED_FLOPS` etc. Are you suggesting exposing an explicit cmake `option()`? I'm fine either way, as it's a pretty "advanced" option that most users don't care about 

Eventually it might be nice to try and auto detect it...

Reply:
good point - you can just pass this in at compile time with `-D` through cmake flags